### PR TITLE
log start/end of each integration test, make test method names unique

### PR DESCRIPTION
**Pull request by rasahner**

### CODE REVIEW

File: integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package io.druid.testing.utils;
```
Comment (fjy) [inline integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java]: header

Comment (rasahner) [inline integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java]: added

File: integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import org.testng.ITestResult;
+import org.testng.TestListenerAdapter;
+
+public class LoggerListener extends TestListenerAdapter
+{
+
+  @Override
+  public void onTestFailure(ITestResult tr)
+  {
+    System.out.println(tr.getName() + "--Test method failed");
```
Comment (fjy) [inline integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java]: can we use a logger instead of System out println?

**Comment by fjy** [comment]
@drcrallen https://travis-ci.org/druid-io/druid/builds/100735941

JDBC namespace extraction still failing

**Comment by fjy** [comment]
@rasahner just to make sure, this PR is based from absolute latest master right?

**Comment by himanshug** [comment]
:+1: after https://github.com/druid-io/druid/pull/2211#discussion_r49035541 is resolved.

File: integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java (hunk 2 above)
Comment (rasahner) [inline integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java]: done

**Comment by rasahner** [comment]
rebased off master

File: integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import com.metamx.common.logger.Logger;
+
+public class LoggerListener extends TestListenerAdapter
+{
+  private static final Logger LOG = new Logger(LoggerListener.class);
+
+  @Override
+  public void onTestFailure(ITestResult tr)
+  {
+    LOG.info(tr.getName() + " -- Test method failed");
```
Comment (himanshug) [inline integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java]: can you use LOG.info("[%s] -- Test method failed", tr.getName())?
applies to other places as well.

Comment (rasahner) [inline integration-tests/src/main/java/io/druid/testing/utils/LoggerListener.java]: done

**Comment by fjy** [comment]
:+1:

**Pull request merged**